src/core/internal/convert.d

```diff
@@ -769,7 +769,7 @@ const(ubyte)[] toUbyte(T)(const ref T val) if (is(T == enum))
     if (__ctfe)
     {
         static if (is(T V == enum)){}
-        return toUbyte(cast(const V) val);
+        return toUbyte(*cast(const V*) &val);
```

Comment:
This change is reported to not be covered by any tests.

Reply:
The coverage report doesn't take into account the `tests` directory. It is covered by the new test added in this PR.